index.html

```diff
@@ -5635,45 +5648,45 @@ <h3>Language and Base Direction</h3>
 
         <pre class="example nohighlight" title="Arabic text with a base direction of right-to-left">
 "title": {
-  "value": "<span class="highlight">HTML و CSS: تصميم و إنشاء مواقع الويب</span>",
-  "lang": "<code>ar</code>"
-  "dir": "<code>rtl</code>"
+  "@value": "<span class="highlight">HTML و CSS: تصميم و إنشاء مواقع الويب</span>",
```

Comment:
You didn't implement the fix here? (It is correct on line 5675)

Reply:
I did in https://github.com/w3c/vc-data-model/pull/1340/commits/0222763398dd0d2d505ad570598e0a10b05aa410

Reply:
Oh. Missed the other line. My bad. Fixing.